Files/Interacts/Interaction.cs

```diff
@@ -109,6 +109,16 @@ public async void OpenInNewWindowItem_Click(object sender, RoutedEventArgs e)
                     await Launcher.LaunchUriAsync(folderUri);
                 }
             }
+            else if (CurrentSourceType == typeof(TilesBrowser))
```

Comment:
Don't copy code from PhotoAlbum "if", you can just use 
`if (CurrentSourceType == typeof(PhotoAlbum) || if (CurrentSourceType == typeof(TilesBrowser)))`

Reply:
These changes have been fixed/removed